doc/whats_new/v1.1.rst

```diff
@@ -287,6 +287,14 @@ Changelog
   `max_iter` and `tol`.
   :pr:`21341` by :user:`Arturo Amor <ArturoAmorQ>`.
 
+:mod:`sklearn.kernel_approximation`
+...................................
+
+- |API| Adds :term:`get_feature_names_out` to the transformers following
+  :mod:`~sklearn.kernel_approximation` module:
+  :class:`~sklearn.kernel_approximation.AdditiveChi2Sampler`.
```

Comment:
I don't understand what you mean by transformers following a module. Maybe you meant the following?

```suggestion
- |API| Adds :term:`get_feature_names_out` to the following transformers 
  of the :mod:`~sklearn.kernel_approximation` module:
  :class:`~sklearn.kernel_approximation.AdditiveChi2Sampler`.
```